packages/@sanity/form-builder/src/inputs/ObjectInput/styles.ts

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components'
+
+export const FieldWrapper = styled.div`
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  min-width: 0;
+`
```

Comment:
This component seems to not be used anymore.

Reply:
Good catch!

Reply:
Fixed in https://github.com/sanity-io/sanity/commit/89d0973e679510bcaa1d7aa573ef1e2e771d35c2